core/aof.go

```diff
@@ -94,19 +95,31 @@ func DumpAllAOF(store *Store) error {
 		aof *AOF
 		err error
 	)
-	if aof, err = NewAOF(config.AOFFile); err != nil {
+	if aof, err = NewAOF(config.TempAOFFile); err != nil {
 		return err
 	}
 	defer aof.Close()
 
-	log.Println("rewriting AOF file at", config.AOFFile)
+	log.Println("rewriting AOF file at", config.TempAOFFile)
 
 	withLocks(func() {
 		for k, obj := range store.store {
 			err = dumpKey(aof, *((*string)(k)), obj)
 		}
 	}, store, WithStoreLock())
 
+	log.Println("flush temp file's data to disk")
+	if err := syscall.Fsync(int(aof.file.Fd())); err != nil {
+		fmt.Println("fsync failed")
```

Comment:
Are these `Println` statements required? If yes shall we use `fmt. Errorf()` instead?